hack/pip-packages/pip-getter/Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM centos:7.9.2009
 
-RUN yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm && \
+RUN yum install -y epel-release && \
```

Comment:
Doesn't this resolve to the same url as before?

Reply:
It should try to download it from mirror list from centos-extra repo.
```
#additional packages that may be useful
[extras]
name=CentOS-$releasever - Extras
mirrorlist=http://mirrorlist.centos.org/?release=$releasever&arch=$basearch&repo=extras&infra=$infra
#baseurl=http://mirror.centos.org/centos/$releasever/extras/$basearch/
gpgcheck=1
gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7
```

Reply:
can you run this in the container and see where it tries to download this from?

Reply:
```
sh-4.2# yum install epel-release
Loaded plugins: fastestmirror, ovl
Loading mirror speeds from cached hostfile
 * base: repos.lax.layerhost.com
 * extras: repos.lax.layerhost.com
 * updates: mirror.hostduplex.com
Resolving Dependencies
--> Running transaction check
---> Package epel-release.noarch 0:7-11 will be installed
--> Finished Dependency Resolution

Dependencies Resolved

======================================================================================================================================
 Package                             Arch                          Version                        Repository                     Size
======================================================================================================================================
Installing:
 epel-release                        noarch                        7-11                           extras                         15 k

Transaction Summary
======================================================================================================================================
Install  1 Package
```

Reply:
I was looking for this 

```
$ docker run --entrypoint=/bin/bash -it centos:7.9.2009
[root@5d6fad256172 /]# yum install epel-release
Loaded plugins: fastestmirror, ovl
Determining fastest mirrors
 * base: coresite.mm.fcix.net
 * extras: mirror.wdc1.us.leaseweb.net
 * updates: mirror.math.princeton.edu
base                                           | 3.6 kB     00:00     
extras                                         | 2.9 kB     00:00     
updates                                        | 2.9 kB     00:00     
(1/4): base/7/x86_64/group_gz                    | 153 kB   00:00     
(2/4): extras/7/x86_64/primary_db                | 250 kB   00:00     
(3/4): base/7/x86_64/primary_db                  | 6.1 MB   00:00     
(4/4): updates/7/x86_64/primary_db               |  17 MB   00:01     
Resolving Dependencies
--> Running transaction check
---> Package epel-release.noarch 0:7-11 will be installed
--> Finished Dependency Resolution

Dependencies Resolved

======================================================================
 Package             Arch          Version        Repository     Size
======================================================================
Installing:
 epel-release        noarch        7-11           extras         15 k

Transaction Summary
======================================================================
Install  1 Package
```

looks like its doing it from extras

Reply:
```
sh-4.2# repoquery -i epel-release

Name        : epel-release
Version     : 7
Release     : 14
Architecture: noarch
Size        : 25728
Packager    : Fedora Project
Group       : System Environment/Base
URL         : http://download.fedoraproject.org/pub/epel
Repository  : epel
Summary     : Extra Packages for Enterprise Linux repository configuration
Source      : epel-release-7-14.src.rpm
Description :
This package contains the Extra Packages for Enterprise Linux (EPEL) repository
GPG key as well as configuration for yum.
```